app/models/cms/loop_html.rb

```diff
@@ -0,0 +1,34 @@
+class Cms::LoopHtml
+  extend SS::Translation
+  include SS::Document
+  include SS::Reference::Site
+  include Cms::SitePermission
+  include Cms::Addon::Html
+
+  set_permission_name "cms_loop_htmls", :edit
+
+  seqid :id
+  field :name, type: String
+  field :description, type: String
+  field :order, type: Integer
+  permit_params :name, :description, :order
+  validates :name, presence: true, length: { maximum: 40 }
+  validates :description, length: { maximum: 400 }
+
+  default_scope -> { order_by(order: 1, name: 1) }
```

Comment:
__[rails_best_practices]__
* default_scope is evil [:ref](http://rails-bestpractices.com/posts/2013/06/15/default_scope-is-evil/)

<!-- Comment by [SideCI](https://sideci.com) -->